### PR TITLE
add the new link for OneCompiler

### LINKED ISSUE #1118: [Add] <OneCompiler in Frontend/Online code editor>
### Site URL

https://onecompiler.com

### Description of the above provided site

I found it hard to make a frontend disign and implement it (in html, css and js) on the go. I tried many websites and plateforms but the problem I faced was irresponsiveness of the plateform with mobile devices and on few the pasting code was not working. Aside being fully **optimised editor and responsive UI**, it also has compatibility with other frameworks and programming languages like **React, Angular, Python, Java** to name a few. It will be one stop editor for anylanguage, anywhere, anydevice. I would love to hear your thoughts on it.

### Category

Frontend

### Sub Category

Online code editor

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Jay9874**
Closes #1118

## Fixes Issue
Closes #1118 

## Changes proposed
Add a new online code editor for html, js, css with compiler for 60+ programming languages and frontend frameworks.
## Screenshots
Screenshot 2023-06-26 at 20 10 40

## Note to reviewers
Should I commit changes after running it locally?

### CODE REVIEW

File: database/frontend/online-code-editors.json [hunk 1]
```diff
@@ -69,6 +69,13 @@
     "category": "frontend",
     "subcategory": "online-code-editors"
   },
+  {
+    "name": "OneCompiler",
+    "description": "Got a new feature for your web app! implement it on mobile, anywhere. It supports over 40 programming languages and frontend frameworks.",
```
Comment (CBID2) [inline database/frontend/online-code-editors.json:74]: [Proofreading]
```suggestion
    "description": "Got a new feature for your web app? Implement OneComplier on mobile and anywhere. It supports over 40 programming languages and frontend frameworks.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Jay9874! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through [Grammarly](https://www.grammarly.com/grammar-check) before making a pull request. Other than that, you're off to a great start! :)

File: database/frontend/online-code-editors.json (hunk 1 above)
Comment (Jay9874) [reply database/frontend/online-code-editors.json:74]: Yes @CBID2, its my mistake. Thanks for correcting it. With your corrections, can we make the description "Got a new feature for your web app? Implement it with OneCompiler on mobile and anywhere. It supports over 60+ programming languages and frontend frameworks."

Comment (CBID2) [reply database/frontend/online-code-editors.json:74]: That's good too @Jay9874! :) It'll tell devs why they should consider using this source.

**Pull request merged**